docs/development/tutorial/README.md

```diff
@@ -54,3 +61,32 @@ It is also often useful to run code formatting:
 ```
 ./gradew spotlessApply
 ```
+
+
+## Running some tests
```

Comment:
Looking at this document, it seems that:
(1) it is a how-to, not a tutorial. How-to assumes a person knows the tools, but just not specifically for TripleA. A tutorial strives to teach, eg: let me teach you how to build java code using gradle, vs 'this is how you use gradle with triplea to do a build, step 1, step 2"

Ergo, this document is misplaced to begin with. More importantly, another implication is that the content should all be very terse and additional examples should only be footnotes. A 'how-to' should read very similar to a cooking recipe and give a concise and single way on how to do something. Hence, identifying this document as a how-to helps clarify what should be presented.

Though last, I think it turns out this doc is a duplicate of:  `how-to/local-build-and-testing.md` .   :facepalm: 
Note how both have similar sections and are both the 'if you are dev, here is how you build, test and run triplea'.

To compensate for this latter point, and if you agree that these two docs are basically the same, I'd suggest we remove the changes from this file and merge them in with `how-to/local-build-and-testing.md` . I believe that would have the benefit of removing some redundancy.

As follow-up, we could then merge in the remaining content from this ('tutorial/README') page into the 'how-to' page and remove the 'tutorial/README' page (and will need to take care to update links)

Reply:
Sounds good. (I wasn't convinced from tutorial/README as it duplicates many things from /how-to), I'll prepare something.

I also don't like the tutorial/pitfalls.md as this is not a tutorial either.

Reply:
It's a good mention about the pitfalls doc, that would seemingly belong somewhere else as well.

Reply:
The thing is, I think it does not fit in any of the four categories (background, how-to, reference, tutorial).
Maybe put it directly under `docs/development`?